week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +46,9 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let removed = arr.splice(index , 1);
```

Comment:
If you read the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) carefully for the `.splice()` array method you'll see that:

> The splice() method changes the contents of an array by removing or replacing existing elements and/or adding new elements

So it does change the original array. I think you'll need to use the `.slice()` method twice instead.